packages/header/package.json

```diff
@@ -1,9 +1,17 @@
 {
   "name": "@development-framework/header",
   "license": "MIT",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "dependencies": {
-    "@development-framework/dm-core": ">=1.0.35",
+    "@development-framework/dm-core": ">=1.0.38",
+    "@development-framework/yaml-view": ">=1.0.4",
+    "@development-framework/explorer": ">=1.0.0",
+    "@development-framework/default-pdf": "^1.0.5",
+    "@development-framework/default-preview": "^1.0.2",
+    "@development-framework/blueprint": "^1.0.5",
+    "@development-framework/mermaid": "^1.0.2",
+    "@development-framework/tabs": "^1.0.3",
+    "@development-framework/form": "^1.0.0",
```

Comment:
Should not be added here.
Instead should be loaded from the dm-core "UiPluginsContext". Similar to what is done in "UiPluginsSelector".